src/stores/wallet/WalletHistoryStore.js

```diff
@@ -27,7 +26,18 @@ export default class {
   @observable perPage = INIT_VALUES.perPage
   @observable page = INIT_VALUES.page
 
-  constructor() {
+  constructor(app) {
+    this.app = app;
+
+    reaction(
+      () => this.app.global.syncBlockNum,
+      () => {
+        if (this.app.ui.location === Routes.WALLET) {
```

Comment:
Does this work at the current version? I don't think the 'hard code' routes bug has been resolved by Alex.